pom.xml

```diff
@@ -44,16 +44,13 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-site-plugin</artifactId>
         </plugin>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-compiler-plugin</artifactId>
-        </plugin>
```

Comment:
should be a different patch? own patch as it is not related, right?

Reply:
Reverted that. I had to include maven-compiler-plugin directly under plugins as well though, to enforce java 8. Since Jenkins is java8+ only anyway.